src/win32/ptpopen.c

```diff
@@ -70,8 +70,7 @@ static int my_pipe(HANDLE *readwrite)
   sa.bInheritHandle = 1;            /* the child must inherit these handles */
   sa.lpSecurityDescriptor = NULL;
 
-  if (! CreatePipe (&readwrite[0],&readwrite[1],&sa,1 << 13))
-  {
+  if (!CreatePipe(&readwrite[0],&readwrite[1],&sa,1 << 13)) {
```

Comment:
Considering this is contrib code, I don't see the point of (in)correcting its brace style to match the rest of libxmp.

Reply:
OK, dropping them in next force-push